### PR TITLE
add webpack

### PR DESCRIPTION
**Pull request by nimaen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,27 @@
 # service-website
+
+Elium's corporate website
+
+----------
+
+## Features overview
+
+The project use **webpack** as its resource bundler.
+It supports :
```
Comment (firehist) [inline README.md]: In english there is no space before `:` :trollface:

File: app/index.html [hunk 2]
```diff
@@ -1,5 +1,6 @@
 <!DOCTYPE html>
-<html lang="en"><head>
+<html lang="en">
+<head>
```
Comment (firehist) [inline app/index.html]: ❤️

File: package.json [hunk 3]
```diff
@@ -3,17 +3,29 @@
   "version": "1.0.0",
   "description": "Elium.tv website",
   "scripts": {
-    "server": "live-server --open=app --watch=app",
+    "clean": "rimraf ./build",
+    "build": "npm run clean && NODE_ENV=development webpack",
+    "dev": "npm run build && NODE_ENV=development webpack-dev-server --progress --colors --hot --content-base ./build/",
```
Comment (firehist) [inline package.json]: Replace w/ `start` to be able to run `npm start` instead of `npm run dev` ?
https://docs.npmjs.com/misc/scripts

**Comment by firehist** [comment]
Except english syntax :trollface: LGTM 🚀

File: app/index.html (hunk 2 above)
Comment (nimaen) [inline app/index.html]: :goat:

File: package.json (hunk 3 above)
Comment (nimaen) [inline package.json]: Yeah sure, will add that option :smile:

File: README.md (hunk 1 above)
Comment (nimaen) [inline README.md]: Derp :dash:

File: README.md [hunk 4]
```diff
@@ -1 +1,27 @@
...
+
+## Start
+
+Init the project by installing the needed dependencies with `npm i`.
+
+## Commands
+
+Build : `npm run build`
+Compile : `npm run compile`
+Build and Watch : `npm run dev`
```
Comment (firehist) [inline README.md]: You need blank line between these to avoid one-liner !
https://github.com/Elium/service-website/blob/feat/webpack/README.md#commands

**Pull request merged**